Sources/TuistGenerator/Mappers/SynthesizedResourceInterfaceProjectMapper.swift

```diff
@@ -135,6 +135,7 @@ public final class SynthesizedResourceInterfaceProjectMapper: ProjectMapping { /
                         parser: resourceSynthesizer.parser,
                         templateString: templateString,
                         name: name,
+                        bundleName: project.options.disableBundleAccessors ? nil : "Bundle.module",
```

Comment:
nit: maybe we can pass the boolean and move this logic to the function?

Reply:
This is the reason behind this extra check here - we ensure the project compilation will not fail in case bundle accessors are disabled. That said, it will most likely not produce the results you had hoped for, so perhaps a lint warning to encourage enabling both options?

Reply:
@danyf90 

My thinking was, we already passed the `name` parameter, so for consistency I've also passed the bundle name 🙂